generator.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+echo "What do you want your project to be called?"
+
+read project_name
+
+mkdir $project_name
```

Comment:
For example, I'd like to prevent people from using dashes, as it breaks the tests.